spec/pages/facility_page.rb

```diff
@@ -1,21 +1,21 @@
 class FacilityPage < ApplicationPage
 
-  FACILITY_PAGE_HEADING = { xpath: "//h1[text()='All facilities']" }.freeze
-  ADD_FACILITY_GROUP_BUTTON = { xpath: "//a[@class='btn btn-sm btn-primary']" }.freeze
-  ORGANISATION_LIST = { xpath: '//h1' }.freeze
-  NEW_FACILITY = { xpath: "//a[text()=' New Facility']" }.freeze
+  FACILITY_PAGE_HEADING = { css: 'h1.page-title' }.freeze
+  ADD_FACILITY_GROUP_BUTTON = { css: 'a.btn.btn-sm.btn-primary' }.freeze
+  ORGANISATION_LIST = { css: 'h2' }.freeze
+  NEW_FACILITY = { css: 'a.btn.btn-default.float-right' }.freeze
```

Comment:
Can we check the text for this button and list instead of the chained css classes?

Should we add custom css classes to help target this better?